manifests/admin.pp

```diff
@@ -31,6 +31,7 @@
     environment => [
       'DJANGO_SETTINGS_MODULE=pulpcore.app.settings',
       "PULP_SETTINGS=${pulp_settings}",
+      'PULP_STATIC_ROOT=/var/lib/pulp/assets/',
```

Comment:
Could you make this a parameter on the `pulpcore` class so we're certain they are in sync?